src/commands/context/spec/context.spec.server.ts

```diff
@@ -117,7 +205,25 @@ describe('sasjs context', () => {
 
   describe('delete', () => {
     it(
-      'should delete a compute context',
+      'should return an error if trying to delete default compute context',
+      async () => {
+        setDefaultContextName()
+
+        const command = new Command(
+          `context delete ${testContextConfig.name} -t targetName`
+        )
+
+        await expect(processContext(command)).resolves.toEqual(
+          defaultContextError('Deleting')
+        )
+
+        restoreTestContextName()
+      },
+      60 * 1000
```

Comment:
It's best to use a `done` callback here and avoid specifying an explicit timeout, since that is already set in the `jest.config.js` file.